table/policy.go

```diff
@@ -1955,6 +1956,54 @@ func NewRouteTypeCondition(c config.RouteType) (*RouteTypeCondition, error) {
 	}, nil
 }
 
+type AfiSafiInCondition struct {
+	afiSafiTypes []config.AfiSafiType
```

Comment:
done, thanks.

Reply:
Did you decide to keep the type `[]config.AfiSafiType`?

Reply:
I've changed GRPC API parts as not to duplicate definitions, but I think from config perspective it's better to use auto-generated `AfiSafiType`, since it's what the openconfig spec says: https://github.com/openconfig/public/blob/master/release/models/bgp/openconfig-bgp-policy.yang#L411

Reply:
Yeah, `config` package should use auto-generated `AfiSafiType`.
I don't have strong will though, I think `table` package can use other type than that of `config` package for the efficiency or so. For example, if we store them as `[]config.AfiSafiType`, we need to convert them into `bgp.RouteFamily` every time when doing `Evaluate()`.

Reply:
Done. `New` will still take config as other constructors do, but internally it uses translated `bgp.RouteFamily`.

Reply:
Thanks!